.github/workflows/CI.yml

```
@@ -219,11 +222,11 @@
         run: |
           cargo test --no-run -p amaru
 
-      - name: Cache Amaru's ledger.db
+      - name: Cache Amaru's ledger.${{ matrix.network }}.db
         id: cache-ledger-db
         uses: actions/cache/restore@v4
         with:
-          path: ./ledger.db
+          path: ./ledger.${{ matrix.network }}.db
```

actionlint (1.7.7)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type {magic: number; name: string}

(expression)